keystore_test.go

```diff
@@ -38,48 +40,31 @@ func TestSetGetMethods(t *testing.T) {
 
 	password := []byte("password")
 
-	if err := ks.SetPrivateKeyEntry(pkeAlias, pke, password); err != nil {
-		t.Fatal(err)
-	}
+	err := ks.SetPrivateKeyEntry(pkeAlias, pke, password)
+	require.NoError(t, err)
 
-	if err := ks.SetTrustedCertificateEntry(tceAlias, tce); err != nil {
-		t.Fatal(err)
-	}
+	err = ks.SetTrustedCertificateEntry(tceAlias, tce)
+	require.NoError(t, err)
 
 	pkeGet, err := ks.GetPrivateKeyEntry(pkeAlias, password)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	chainGet, err := ks.GetPrivateKeyEntryCertificateChain(pkeAlias)
-	if err != nil {
-		t.Fatal(err)
-	}
+	require.NoError(t, err)
 
 	tceGet, err := ks.GetTrustedCertificateEntry(tceAlias)
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	if !reflect.DeepEqual(pke, pkeGet) {
-		t.Fatal("private key entries not equal")
-	}
+	require.NoError(t, err)
 
-	if !reflect.DeepEqual(pke.CertificateChain, chainGet) {
-		t.Fatal("certificate chains of private key entries are not equal")
-	}
+	assert.True(t, reflect.DeepEqual(pke, pkeGet), "private key entries not equal")
+	assert.True(t, reflect.DeepEqual(pke.CertificateChain, chainGet),
+		"certificate chains of private key entries are not equal")
+	assert.True(t, reflect.DeepEqual(tce, tceGet), "private key entries not equal")
```

Comment:
Please consider using assert.Equal and put expected and actual arguments in a correct order.